### PR TITLE
docs: Markdown linting and typo

### PR DESCRIPTION
**Pull request by shyamkumaryadav**
#### Description of Change

doc typo fix MD053

#### Checklist

- [x] `npm test` passes
- [x] relevant documentation is changed or added
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: no-notes

### CODE REVIEW

File: docs/tutorial/electron-versioning.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ There are several major changes from our 1.x strategy outlined below. Each chang
 2. Introduction of semver-compliant `-beta` tags
 3. Introduction of [conventional commit messages](https://conventionalcommits.org/)
 4. Well-defined stabilization branches
-5. The `main` branch is versionless; only stabilization branches contain version information
+5. The `main` branch is version less; only stabilization branches contain version information
```
Comment (codebytere) [inline docs/tutorial/electron-versioning.md:25]: This was intended - please restore it.

File: docs/tutorial/electron-versioning.md [hunk 2]
```diff
@@ -143,7 +143,7 @@ The `electron/electron` repository also enforces squash merging, so you only nee
 
 ## Historical versioning (Electron 1.X)
 
-Electron versions *< 2.0* did not conform to the [SemVer](https://semver.org) spec: major versions corresponded to end-user API changes, minor versions corresponded to Chromium major releases, and patch versions corresponded to new features and bug fixes. While convenient for developers merging features, it creates problems for developers of client-facing applications. The QA testing cycles of major apps like Slack, Teams, Skype, VS Code, and GitHub Desktop can be lengthy and stability is a highly desired outcome. There is a high risk in adopting new features while trying to absorb bug fixes.
```
Comment (codebytere) [inline docs/tutorial/electron-versioning.md:146]: Please restore this as well.

File: docs/tutorial/electron-versioning.md (hunk 1 above)
Comment (shyamkumaryadav) [reply docs/tutorial/electron-versioning.md]: done

File: docs/tutorial/electron-versioning.md (hunk 2 above)
Comment (shyamkumaryadav) [reply docs/tutorial/electron-versioning.md]: done

**Review by jkleinsc — APPROVED** [review]

**Comment by jkleinsc** [comment]
Merging as CI failure unrelated to PR change.

**Pull request merged**